### PR TITLE
new article for auth error resolution

### PR DESCRIPTION
**Pull request by davidmu1**

### CODE REVIEW

File: concepts/resolve-auth-errors.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ms.prod: "microsoft-identity-platform"
+---
+
+# Resolve Microsoft Graph authorization errors
+
+Authorization errors can be a result of several different issues, most of which generate a 401 or 403 error. For example, the following can all lead to authorization errors:
+
+* Incorrect [access token acquisition flows](https://docs.microsoft.com/azure/active-directory/develop/active-directory-authentication-scenarios)
+* Poorly configured [permission scopes](https://docs.microsoft.com/azure/active-directory/develop/active-directory-v2-scopes)
+* Lack of [consent](https://docs.microsoft.com/azure/active-directory/develop/active-directory-devhowto-multi-tenant-overview#understanding-user-and-admin-consent)
```
Comment (dkershaw10) [inline concepts/resolve-auth-errors.md]: Another one:  Lack of permissions for the signed-in user in interactive flows (link to this topic: https://docs.microsoft.com/en-us/graph/auth-v2-user?context=graph%2Fapi%2F1.0&view=graph-rest-1.0).  This is generally due to the user not being in a privileged enough role - the app may have access, but the user doesn't.

File: concepts/resolve-auth-errors.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+title: "Resolve Microsoft Graph authorization errors"
+description: "How to resolve Microsoft Graph 401 and 403 authorization errors."
+author: "davidmu1"
+localization_priority: Priority
+ms.prod: "microsoft-identity-platform"
+---
+
+# Resolve Microsoft Graph authorization errors
+
+Authorization errors can be a result of several different issues, most of which generate a 401 or 403 error. For example, the following can all lead to authorization errors:
```
Comment (dkershaw10) [inline concepts/resolve-auth-errors.md]: I think it's most of which generate a 403 error (with a few exceptions).

Comment (dkershaw10) [inline concepts/resolve-auth-errors.md]: I think it's most of which generate a 403 error (with a few exceptions).

**Comment by Lauragra** [comment]
@davidmu1 , checking in here, are we ready to publish this article?

**Pull request merged**